.github/workflows/issue-create.yml

```diff
@@ -0,0 +1,34 @@
+on:
+  # Trigger analysis when pushing changes to main branch
+  push:
+    paths:
+      - 'data/**'
+    branches:
+      - master
+name: Git Issue Create On EU Data Update
+jobs:
+  gitIssueCreation:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        repo:
+          - CMSgov/qpp-eligibility
+          - CMSgov/epcs-etl
+    steps:
+      - name: Checkout Codebase
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 2
+
+      - name: Find changed files
+        id: getfile
+        run: |
+          echo "::set-output name=files::$(git diff-tree --no-commit-id --name-only -r ${{ github.sha }} | grep -e '.*\.csv$' | xargs)"
+
+      - name: Show files changed
+        run: |
+          echo ${{ steps.getfile.outputs.files }}
+
+      - uses: .github/ISSUE_TEMPLATE/create_issue_epcs_eligibility.md
```

Comment:
the file is not an action right.  wondering, how this will work.  did you try creating an issue with this code?

Reply:
I can test this on merge. Have updated it use action.